{
  "id": "ch_CSY11CK7T_2024-03-25_1711376122.264009_thread",
  "type": "channel",
  "channel_name": "discuss-frontend",
  "conversation_type": "thread",
  "participants": [
    "Anil",
    "Bala"
  ],
  "messages": [
    {
      "sender": "Bala",
      "user_id": "U045MQSQYKB",
      "message": "Hey @Anil, do we have an update on this case? For now, the client had been given a workaround to fetch the chunk logs from `system/global`.",
      "time": "07:15",
      "timestamp": "1711376122.264009",
      "is_reply": false
    },
    {
      "sender": "Anil",
      "user_id": "U01BH8XBR55",
      "message": "@Bala looks like this is a rare error, some of the logs are corrupted and not proper JSONs.\nhttps://instabase.atlassian.net/browse/FLOW-8830",
      "time": "09:30",
      "timestamp": "1711384213.917959",
      "is_reply": true
    },
    {
      "sender": "Anil",
      "user_id": "U01BH8XBR55",
      "message": "Created a ticket for it. I donâ€™t have the bandwidth atm to fix this - would advise customer to use the workaround. Looks like it is not happening anymore.",
      "time": "09:30",
      "timestamp": "1711384242.616429",
      "is_reply": true
    }
  ],
  "metadata": {
    "channel_id": "CSY11CK7T",
    "channel_name": "discuss-frontend",
    "date_file": "2024-03-25.json",
    "message_count": 3,
    "start_time": "1711376122.264009",
    "end_time": "1711384242.616429",
    "is_thread": true
  }
}